lib/helm-util.js

```diff
@@ -10,22 +10,15 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
-var __importStar = (this && this.__importStar) || function (mod) {
-    if (mod && mod.__esModule) return mod;
-    var result = {};
-    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
-    result["default"] = mod;
-    return result;
-};
 Object.defineProperty(exports, "__esModule", { value: true });
-const os = __importStar(require("os"));
-const path = __importStar(require("path"));
-const util = __importStar(require("util"));
-const fs = __importStar(require("fs"));
+const os = require("os");
+const path = require("path");
+const util = require("util");
+const fs = require("fs");
```

Comment:
Were these changes auto generated on build? Also, have you tested all the final changes? If everything's done, I'll merge the PR.
CC: @ajinkya599

Reply:
yes they were auto generated. ive tested it. But baking with kompose was failing for some reason (even using azure/k8s-bake@v1), need to look at that. Ajinkya can have a look at the L0 part of this, I still need to change repo [here](https://github.com/sundargs2000/k8s-bake/blob/7fc7fc11b1124768cffd7c7f580fbe4e9010ac5e/.github/workflows/TriggerIntegrationTests.sh#L25) to Azure. Jyotsna now has write access to this repo so she can merge after that.